financial-services/1.10/modules/ROOT/pages/upgrade-notes.adoc

```diff
@@ -0,0 +1,148 @@
+
+=== Core Banking Limitations
+
+It is important to note that the Jack Henry SilverLake provider does not include support for all of the functionality that was originally implemented by the generic Core Banking System APIs. This may cause exisiting test cases and demo scripts to fail under certain conditions - please consult the documentation for the SilverLake System APIs for more details.
+
+== Release 1.5 to Release 1.6
+
+There were no breaking changes introduced by this release. The existing Salesforce Customers System API was enhanced to support the posting of messages to chatter feeds; all other APIs included in this release were net new.
+
+== Release 1.4 to Release 1.5
+
+The following changes introduced in this release may impact existing deployments:
+
+* The publishing of updates to customer profiles (including individuals and organizations) has been streamlined to use only one set of Anypoint MQ destinations for parties and another set for party roles. Upgrading will require the new destinations to be created before redeploying any affected assets.
+* The DocuSign Experience and System APIs have been refactored to make them more generic. Assets related to the processing of Loan Applications have been updated accordingly and will need to be redeployed. This includes the Bank Loans Process API and the Loan Origination System API.
+
+== Release 1.3 to Release 1.4
+
+The existing customer sync process was augmented yet again to support the Postal Address Change use case, which will have an impact on existing deployments of these components. There is also a new, cleaner version of the CIM Market library included in this release.
+
+=== Change Summary
+
+The following changes may have an impact on existing deployments:
+
+* The CIM libraries were upgraded to new versions containing better examples and simplified (but more complete) JSON schemas.
+* The Notifications Process API is now referenced as a shared service.
+* Support for additional Salesforce FSC record types has been added.
+* The ID Graph LWC has been updated.
+
+=== Upgrade Steps
+
+If a release earlier than 1.3, incorporate the previous upgrade steps into the following ones.
+
+. Apply the latest configuration changes to your Salesforce Financial Services Cloud instance as documented in the customization guide.
+. Deploy the latest ID Graph LWC asset to your target Salesforce org(s).
+. Apply the latest schema changes to the Core Banking and MDM databases.
+** This may require some updates to existing records.
+. Deploy the latest customer sync components included in this release.
+
+=== Verification Steps
+
+* Create a new customer in Salesforce FSC, ensure it is replicated to Core Banking and MDM.
```

Comment:
This isn't a complete sentence as written. Either "To create ..." or "FSC; ensure"